testsuite/tests/typing-local/let_mutable.ml

```diff
@@ -568,6 +568,45 @@ Error: This expression has type "int" but an expression was expected of type
   Hint: Did you mean "3."?
 |}]
 
+(* Test 18.1: unmutated mutable variable warning *)
+let x_18_1 =
+  let mutable x = 3 in x + 1
+;;
+[%%expect{|
+Line 2, characters 14-15:
+2 |   let mutable x = 3 in x + 1
+                  ^
+Warning 186 [unused-mutable]: variable x does not need to be mutable.
```

Comment:
I think that, just to help people pattern match, we should rephrase this to be closer to warning 69:
```
File "/home/ccasinghino/tmp/mut.ml", line 6, characters 13-28:
6 |   type t = { mutable x : int }
                 ^^^^^^^^^^^^^^^
Warning 69 [unused-field]: mutable record field x is never mutated.
```